interface/usergroup/usergroup_admin.php

```diff
@@ -342,154 +343,31 @@
-                    for ($iter = 0;$row = sqlFetchArray($res);$iter++)
-                        $result4[$iter] = $row;
-                    foreach ($result4 as $iter) {
-                        if ($iter{"authorized"}) {
-                            $iter{"authorized"} = xl('yes');
-                        } else {
-                            $iter{"authorized"} = "";
-                        }
-                        print "<tr>
-                            <td><b><a href='user_admin.php?id=" . $iter{"id"} .
-                            "' class='iframe_medium' onclick='top.restoreSession()'>" . $iter{"username"} . "</a></b>" ."&nbsp;</td>
-                            <td>" . attr($iter{"fname"}) . ' ' . attr($iter{"lname"}) ."&nbsp;</td>
-                            <td>" . attr($iter{"info"}) . "&nbsp;</td>
-                            <td align='left'><span>" .$iter{"authorized"} . "&nbsp;</td>";
-                        print "<td><!--<a href='usergroup_admin.php?mode=delete&id=" . $iter{"id"} .
-                            "' class='link_submit'>[Delete]</a>--></td>";
-                        print "</tr>\n";
-                    }
-                    ?>
-                </tbody>
-            </table>
-            <?php
-            if (empty($GLOBALS['disable_non_default_groups'])) {
-                $res = sqlStatement("select * from groups order by name");
-                for ($iter = 0;$row = sqlFetchArray($res);$iter++)
-                    $result5[$iter] = $row;
-
-                foreach ($result5 as $iter) {
-                    $grouplist{$iter{"name"}} .= $iter{"user"} .
-                        "(<a class='link_submit' href='usergroup_admin.php?mode=delete_group&id=" .
-                        $iter{"id"} . "' onclick='top.restoreSession()'>Remove</a>), ";
-                }
-
-                foreach ($grouplist as $groupname => $list) {
-                    print "<span class='bold'>" . $groupname . "</span><br>\n<span>" .
-                        substr($list,0,strlen($list)-2) . "</span><br>\n";
-                }
-            }
-            ?>
-        </div>
-    </div>
-</div>
-<script language="JavaScript">
-<?php
-  if ($alertmsg = trim($alertmsg)) {
-    echo "alert('$alertmsg');\n";
-  }
-?>
-</script>
-</body>
-</html>
+$request = Request::createFromGlobals();
+
+$users = function($active = 1) {
+    $sql = "SELECT * FROM users WHERE username IS NOT NULL %s";
+    if ($active == 1) {
+        $sql = sprintf($sql, "AND active = 1");
+    } else if ($active == 0) {
+        $sql = sprintf($sql, "");
+    }
+    $res = sqlStatement($sql);
+    $return = [];
+    while ($row = sqlFetchArray($res)) {
+        $row['fullName'] = sprintf("%s, %s", $row['lname'], $row['fname']);
+        $row['authorized'] = ($row['authorized'] == 1) ? xlt("Yes") : xlt("No");
+        $return[] = $row;
+    }
+    return $return;
+};
+
+$viewVars = [
+    "title" => xlt("User / Group"),
```

Comment:
Just use xl since escaping will happen in twig (note this makes sense since twig is where the html output happens).